contracts/gov/ERC20/ERC20Sale.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "../../interfaces/gov/ERC20/IERC20Sale.sol";
+
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Capped.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Pausable.sol";
+
+contract ERC20Sale is IERC20Sale, ERC20Capped, ERC20Pausable {
+    address public govAddress;
+
+    modifier onlyGov() {
+        require(msg.sender == govAddress, "ERC20Sale: not a Gov contract");
+        _;
+    }
+
+    constructor(
```

Comment:
Let's unpause the contract at the end of the constructor

Reply:
discussed